source/highlights.rst

```diff
@@ -56,7 +56,7 @@ Develop with NanoVer
 **Contribute to the code** or **develop your own custom applications**.
 
 * `Get the source code for the server <https://github.com/IRL2/nanover-protocol>`_
-* `Get the source code of the VR client <https://github.com/IRL2/nanover-imd>`_
+* `Get the source code of the VR client <https://github.com/IRL2/nanover-imd-vr>`_
 
 NanoVer is distributed under the `MIT <https://github.com/IRL2/nanover-protocol/blob/main/LICENSE>`_ license.
```

Comment:
```suggestion
NanoVer is distributed under the `MIT <https://github.com/IRL2/nanover-server-py/blob/main/LICENSE>`_ license.
```

Reply:
This change is also being addressed in https://github.com/IRL2/nanover-docs/pull/132 and is out of scope of this PR, so I'm gonna reject it here